tests/protos/test_message_pb2.py

```diff
@@ -2,544 +2,932 @@
 # source: test_message.proto
```

Comment:
We don't need these changes, right?

Reply:
@WeichenXu123 Looks like you applied black formatting on this file.